Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -369,28 +369,32 @@ int RGroupDecomposition::add(const ROMol &inmol) {
     return -2;
   }
 
+  const bool prune = true;
+  if (prune) {
```

Comment:
There is a rationale here. The current code always passes `prune = true` to `process(0`. To make code more readable, I define a `const` as we do elsewhere. Regarding the check: I know that the check at the moment is just a waste of CPU cycles, but I would like that the code still behaves if someone for some reason decides to set `prune` to `false` in his own fork of the code rather then just assuming that prune is always `true`.

Reply:
maybe just a quick comment to that effect?
```suggestion
  const bool prune = true;
  // in case the value ends up being changed in a future version of the code:
  if (prune) {
```